AsyncDisplayKit/Layout/ASLayoutSpec.mm

```diff
@@ -90,6 +94,21 @@ - (ASLayout *)measureWithSizeRange:(ASSizeRange)constrainedSize
     //}
 
     id<ASLayoutable> finalLayoutable = [child finalLayoutable];
+    
+#if LAYOUT_VALIDATION
+    if (self.shouldValidate && finalLayoutable.shouldValidate) {
+      for (ASLayoutableValidationBlock validatorBlock in [self layoutableValidatorBlocks]) {
+        NSError *error = nil;
+        BOOL success = validatorBlock(finalLayoutable, &error);
+        
+        if (error != nil) {
+          ASDisplayNodeCAssert(success, error.localizedDescription);
```

Comment:
I think it would be valuable to modify the error message with a way for them to suppress validation if it becomes bothersome in a particular case. E.g. append to the end of the error message `To suppress this issue, set 'shouldValidate=NO on <ASStackLayoutSpec: 0xFFFFFFFF>`. Man we should really support debug names on these specs.